frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -114,7 +103,12 @@ export function SurveyForm({ id }: { id: string }): JSX.Element {
                 <Field
                     name="linked_flag_id"
                     label="Link feature flag (optional)"
-                    info={<>Feature you want to connect this survey to.</>}
+                    info={
+                        <>
+                            Connecting to a feature flag will automatically enable this survey for everyone in the
+                            feature flag.
```

Comment:
I'm not clear on what this means. If it's a boolean feature flag, is it enabled for people with a `true` value? I'd assume so. But what if it's a multivariate flag? Anyone who matches any of the variants?

Reply:
No multivariate flags functionality on this yet since you can't choose which variant to show the survey to or not, so if you link it to a multivariate flag it'll just be enabled to anyone who has any variant for that flag